tools/build_config/Jenkinsfile

```
@@ -190,6 +190,28 @@ pipeline {
       }
     }
 
+    stage('Configure') {
+      if (isUnix()) {
+        sh '''
+          export PATH=${PATH}:~/.local/bin &&
+          module load cmake/\$CMAKE_VERSION &&
+          module load matlab/\$MATLAB_VERSION &&
+          module load gcc/\$GCC_VERSION &&
+          module load cppcheck/\$CPPCHECK_VERSION &&
+          ./tools/build_config/build.sh \
+            --cmake_flags \"-DHorace_RELEASE_TYPE=\$RELEASE_TYPE\" \
+            --matlab_release \$MATLAB_VERSION \
+            --print_versions --configure
+          '''
+      } else {
+        powershell '''
+           ./tools/build_config/build.ps1 -print_versions -configure
```

Could you split each flag onto a separate line? Just makes it easier to visually parse
```suggestion
           ./tools/build_config/build.ps1 \
             -print_versions \ 
             -configure \
```

(Similar for the equivalent bash commands)